### ISSUE TITLE
avocado list does not consider 'test_dir' for shell scripts

### ISSUE THREAD

**Issue by narasimhan-v** [avocado-framework/avocado#2984]
**Conf file:**
```
[datadir.paths]
base_dir = /root/avocado-fvt-wrapper
test_dir = /root/avocado-fvt-wrapper/tests
data_dir = /root/avocado-fvt-wrapper/data
logs_dir = /root/avocado-fvt-wrapper/results

[sysinfo.collect]
enabled = True
profiler = True
per_test = True

```


**SHELL:**
```
# avocado list -V avocado-misc-tests/io/net/ethtool/ethtool_tests.sh
Test discovery plugin <avocado.core.loader.FileLoader object at 0x3fff85ad03d0> failed: _make_simple_or_broken_test() takes exactly 4 arguments (3 given)
Type Test                                                                                                                                                                                                                                                                    Tag(s)
!VT  avocado-misc-tests/io/net/ethtool/ethtool_tests.sh: Command 'kvm' could not be found in any of the PATH dirs: ['/usr/lib64/qt-3.3/bin', '/bin', '/sbin', '/usr/bin', '/usr/local/sbin', '/usr/libexec', '/opt/ibutils/bin', '/root/bin', '/usr/sbin', '/usr/local/bin']

TEST TYPES SUMMARY
==================
!VT: 1
ACCESS_DENIED: 0
BROKEN_SYMLINK: 0
EXTERNAL: 0
INSTRUMENTED: 0
MISSING: 0
NOT_A_TEST: 0
PYUNITTEST: 0
SIMPLE: 0
VT: 0
```


**PYTHON:**
```
# avocado list -V avocado-misc-tests/io/net/net_data.py
Type         Test                                                                                                                                                                                                                                                       Tag(s)
INSTRUMENTED avocado-misc-tests/io/net/net_data.py:NetDataTest.teststatistics
INSTRUMENTED avocado-misc-tests/io/net/net_data.py:NetDataTest.testbigping
INSTRUMENTED avocado-misc-tests/io/net/net_data.py:NetDataTest.testgro
INSTRUMENTED avocado-misc-tests/io/net/net_data.py:NetDataTest.testlro
INSTRUMENTED avocado-misc-tests/io/net/net_data.py:NetDataTest.testinterface
!VT          avocado-misc-tests/io/net/net_data.py: Command 'kvm' could not be found in any of the PATH dirs: ['/usr/lib64/qt-3.3/bin', '/bin', '/sbin', '/usr/bin', '/usr/local/sbin', '/usr/libexec', '/opt/ibutils/bin', '/root/bin', '/usr/sbin', '/usr/local/bin']

TEST TYPES SUMMARY
==================
!VT: 1
ACCESS_DENIED: 0
BROKEN_SYMLINK: 0
EXTERNAL: 0
INSTRUMENTED: 5
MISSING: 0
NOT_A_TEST: 0
PYUNITTEST: 0
SIMPLE: 0
VT: 0
```

**Comment by narasimhan-v** [comment]
This is a recent behaviour change, not seen on earlier commits.

**Comment by narasimhan-v** [comment]
@ldoktor @clebergnu

**Comment by ldoktor** [comment]
Hello @narasimhan-v, it looks like you're using older Avocado, this was fixed in 6286638d2a9db37f1b37e7100ed73c3eb1383817 anyway can you double-check?

**Comment by clebergnu** [comment]
@narasimhan-v did you get the chance to double check the suggestion from @ldoktor ?

**Comment by clebergnu** [comment]
Closing as this is, to the best of our knowledge, already fixed in 6286638d2a9db37f1b37e7100ed73c3eb1383817
